.github/workflows/codeql-analysis.yml

```diff
@@ -37,10 +37,15 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v3
+      
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libexpat1-dev zlib1g-dev libbrotli-dev
```

Comment:
I wonder how this worked before! expat and zlib are not new.

Reply:
They're installed by default!

Reply:
Another option would be to copy this:
 
https://github.com/Exiv2/exiv2/blob/b90426dda95c7b3f2824b6e137bf9f33820d36b4/.github/workflows/on_PR_linux_fuzz.yml#L26

Reply:
I contemplated that, but seems to be overkill with all the gtest building etc.